service/raft/raft_group0.cc

```diff
@@ -183,6 +183,11 @@ raft_group0::do_discover_group0(raft::server_address my_addr) {
     }
 }
 
+future<> raft_group0::abort() {
+    return _shutdown_gate.close();
```

Comment:
Please, note, that the `_abort_source` can be left ... unrequested in case startup rollback is happenning. Is that OK from group0 code POV?

Reply:
It is used in two one-second sleeps, so yes.